### DISCUSSION TITLE
Is it possible to rename test names when shown in the cocotb regression table displayed at the end of the testbench execution?

### DISCUSSION THREAD

**Discussion by mvsoliveira** [cocotb/cocotb#4170] [Q&A]
Hi,

Is it possible to print inteligible names in the cocotb regression table displayed at the end of the testbench execution? 

Normally I use the cocotb.regression TestFactory to generate tests featuring a combination of different parameters. It would be useful to have list the actual value of the parameters being used for each test in the respective table.

**Comment by ktbarrett** [comment]
`cocotb.parametrize` exists to do exactly that. It should be available in 2.0 and is already available on the master branch.
